src/components/Card.js

```diff
@@ -1,9 +1,22 @@
+import { api } from "../pages/index.js";
+import { deleteConfirmPopup } from "../pages/index.js";
```

Comment:
```js

import { api } from "../pages/index.js";
import { deleteConfirmPopup } from "../pages/index.js";



```

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.

Reply:
Corrected